lib/helmsman/cluster/cluster.ex

```diff
@@ -10,7 +10,13 @@ defmodule Helmsman.Cluster do
     Enum.map(cluster_configs, &from_config/1)
   end
 
+  def determine_advertised_brokers(configured_brokers) do
+    KafkaEx.create_worker(:worker_foo, [uris: configured_brokers, consumer_group: "meh"])
```

Comment:
You can set `:no_consumer_group` IIRC.  Also we might want to give it a meaningful name, like `:helmsman_probe` or something?